plugins/modules/docker_image.py

```diff
@@ -528,12 +536,14 @@ def absent(self):
             self.results['image']['state'] = 'Deleted'
 
     def archive_image(self, name, tag):
-        '''
+        """
```

Comment:
Please change this back to `'''`, and change the other docstring to `'''` as well.

Reply:
Done.